TARGETS

```diff
@@ -67,6 +67,16 @@ is_opt_mode = build_mode.startswith("opt")
 if is_opt_mode:
     rocksdb_compiler_flags.append("-DNDEBUG")
 
+default_allocator = read_config("fbcode", "default_allocator")
+
+sanitizer = read_config("fbcode", "sanitizer")
+
+# Let RocksDB aware of jemalloc existence.
+# Do not enable it if sanitizer presents.
+if default_allocator.startswith("jemalloc") and sanitizer == "":
```

Comment:
Will it always be possible to check allocator type with `startswith`?

Reply:
Do you mean `default_allocator` may not be a string? I'm assuming it will always be a string. Not sure what's best practice for python typing.

Reply:
I mean is it possible that a future jemalloc-based allocator has the name that does NOT start with 'jemalloc'? For example, 'the_new_jemalloc'.

Reply:
So far I'm seeing both "jemalloc" and "jemalloc_debug" in fbcode/mode. I'm a bit hand-waving by just checking the prefix, not doing `if default_allocator in ["jemalloc", "jemalloc_debug"]` but I definitely don't want `the_new_jemalloc` go through because I don't know what's its behavior.